packages/cli/src/load-nodes-and-credentials.ts

```diff
@@ -307,22 +307,28 @@ export class LoadNodesAndCredentials {
 
 		for (const loader of Object.values(this.loaders)) {
 			// list of node & credential types that will be sent to the frontend
-			const { known, types, directory } = loader;
-			this.types.nodes = this.types.nodes.concat(types.nodes);
+			const { known, types, directory, packageName } = loader;
+			this.types.nodes = this.types.nodes.concat(
+				types.nodes.map(({ name, ...rest }) => ({
+					...rest,
+					// TODO: check if CUSTOM nodes should also be prefixed or not
+					name: `${packageName}.${name}`,
+				})),
+			);
 			this.types.credentials = this.types.credentials.concat(types.credentials);
 
 			// Nodes and credentials that have been loaded immediately
-			for (const nodeTypeName in loader.nodeTypes) {
-				this.loaded.nodes[nodeTypeName] = loader.nodeTypes[nodeTypeName];
+			for (const type in loader.nodeTypes) {
```

Comment:
nit: IMHO this was more readable before. Type is very ambiguous name (is it an object or string or what?), where as nodeTypeName was explicit. Or could also be `typeName` if you want to make it shorter